authors/apps/articles/serializers.py

```diff
@@ -48,6 +50,14 @@ def get_favorites_count(self, instance):
         """Return the number of users who have favourited the atricle."""
         return instance.favourited.count()
 
+    def get_share_urls(self, instance):
```

Comment:
I noticed that other SerializerMethodFields have a `method_name` parameter. How is this `get_share_urls` method linked to its SerializerMethodField?

Reply:
Got it 👍